x-pack/plugins/osquery/public/action_results/action_results_summary.tsx

```diff
@@ -236,30 +170,19 @@ const ActionResultsSummaryComponent: React.FC<ActionResultsSummaryProps> = ({
     []
   );
 
-  return (
-    <>
-      <EuiFlexGroup>
-        <EuiFlexItem grow={false}>
-          <StyledEuiCard title="" description="" textAlign="left">
-            {!expired && notRespondedCount ? <EuiProgress size="xs" position="absolute" /> : null}
-            <EuiDescriptionList
-              compressed
-              textStyle="reverse"
-              type="responsiveColumn"
-              listItems={listItems}
-            />
-          </StyledEuiCard>
-        </EuiFlexItem>
-      </EuiFlexGroup>
+  useEffect(() => {
+    setIsLive(
+      // @ts-expect-error update types
+      logsResults?.rawResponse.aggregations?.unique_agents.value !== agentIds?.length ?? !expired
```

Comment:
`??` -> `||` here as well

Reply:
it was intended to use `??` because if left side returns `false` we want to use this value instead of taking the right side vlaue